api/envoy/config/filter/http/jwt_authn/v2alpha/config.proto

```diff
@@ -183,33 +183,67 @@ message JwtHeader {
   string value_prefix = 2;
 }
 
-// This is the Envoy HTTP filter config for JWT authentication.
-// [#not-implemented-hide:]
-message JwtAuthentication {
-  // List of JWT rules to valide.
-  repeated JwtRule rules = 1;
+// This message specifies a Jwt requirement.
+//
+// .. code-block:: yaml
+//
+//    required_issuers: [jwt_issuer1, jwt_issuer2]
+//
+// PerRouteConfig for jwt_authn filter uses this message.
+message JwtRequirement {
+  // Specify list of required Jwt issuers. If list is empty,  Jwt is NOT required.
+  repeated string required_issuers = 1;
 
+  // It only applies when required_issuers field is not empty.
   // If true, the request is allowed if JWT is missing or JWT verification fails.
   // Default is false, a request without JWT or failed JWT verification is not allowed.
   bool allow_missing_or_failed = 2;
+}
 
-  // This field lists the patterns allowed to bypass JWT verification. This only applies when
-  // `allow_missing_or_failed_jwt` is false. Under this config, if a request doesn't have JWT, it
-  // will be rejected. But some requests still needed to be forwarded without JWT, such as OPTIONS
-  // for CORS and some health checking paths.
+// This message specifies a Jwt requirement for a specific Route condition.
+// For example:
+//
+// .. code-block:: yaml
+//
+//    - match:
+//         prefix: "/healthz"
+//      requirement:
+//         required_issuers: []
+//    - match:
+//         prefix: "/"
+//      requirement:
+//         required_issuers: [jwt_issuer1]
+//         allow_missing_or_failed: true
+//
+message RouteRequirement {
+  // The route matching parameter.  Only match is satisfied, the other fields will apply.
   //
-  // Examples: bypass all CORS options requests
+  // For example: following match will match all requests.
   //
   // .. code-block:: yaml
   //
-  //   bypass:
-  //   - headers:
-  //     - name: :method
-  //       value: OPTIONS
-  //   - headers:
-  //     - name: :path
-  //       regex_match: /.*
+  //    match:
+  //       prefix: "/"
   //
+  envoy.api.v2.route.RouteMatch match = 1 [(validate.rules).message.required = true];
+
+  // Specifies the Jwt requiement for matched routes.
+  JwtRequirement requirement = 2;
+}
+
+// This is the Envoy HTTP filter config for JWT authentication.
+// [#not-implemented-hide:]
+message JwtAuthentication {
+  // List of JWT rules to valide.
+  repeated JwtRule rules = 1;
+
+  // DEPRECATED: Please use "routes" field. if "routes" field is not empty,
+  // this field will not be used.
+  // If true, the request is allowed if JWT is missing or JWT verification fails.
+  // Default is false, a request without JWT or failed JWT verification is not allowed.
+  bool allow_missing_or_failed = 2;
```

Comment:
Just delete this instead of `DEPRECATED` since no one is using this? same below.

Reply:
Unfortunately, Istio 0.8 is using this one.  I am pretty sure nobody is using  the "bypass" field.

Reply:
We actually do
https://github.com/istio/istio/blob/0214aee9cb9c75afa59e83364c7acd52afa01656/pilot/pkg/networking/plugin/authn/authentication.go#L128